src/vs/workbench/contrib/interactive/browser/interactive.contribution.ts

```diff
@@ -461,15 +461,40 @@ registerAction2(class extends Action2 {
 				}
 			],
 			icon: icons.executeIcon,
-			f1: false
+			f1: false,
+			description: {
+				description: 'Execute the Contents of the Input Box',
+				args: [
+					{
+						name: 'resource',
+						description: 'Interactive resource Uri',
+						isOptional: true
+					}
+				]
+			}
 		});
 	}
 
-	async run(accessor: ServicesAccessor): Promise<void> {
+	async run(accessor: ServicesAccessor, context?: URI): Promise<void> {
```

Comment:
Since the `context` can come from extension host, it's not guaranteed to be a `URI`. We may want to use `UriComponents`.